docs/reference/search-application/apis/put-search-application.asciidoc

```diff
@@ -206,4 +184,45 @@ PUT _application/search_application/my-app
   }
 }
 ----
-// TEST[skip:TBD]
+
+When the above `dictionary` parameter is specified, the <<search-application-search, search application search>> API will perform the following parameter validation:
+
+* It accepts only the `query_string` and `default_field` parameters
+* It verifies that `query_string` and `default_field` are both strings
+* It accepts `default_field` only if it takes the values `title` or `description`
+
+If the parameters are not valid, the the <<search-application-search, search application search>> API will return an error.
+[source,console]
+----
+POST _application/search_application/my-app/_search
+{
+  "params": {
+    "default_field": "author",
+    "query_string": "Jane"
+  }
+}
+----
+// TEST[catch:bad_request]
+
+In the above example, the value of the `default_field` parameter is not valid, therefore Elasticsearch will return an error:
+
+[source,JSON]
+----
+{
+  "error": {
+    "root_cause": [
+      {
+        "type": "validation_exception",
+        "reason": 'Validation Failed: 1: $.default_field: does not have a value in the enumeration [title, description];',
+        "stack_trace": ...
+      }
+    ],
+    "type": "validation_exception",
+    "reason": 'Validation Failed: 1: $.default_field: does not have a value in the enumeration [title, description];',
+    "stack_trace": ...
+  },
+  "status": 400
+}
+----
+// TESTRESPONSE[s/"stack_trace": \.\.\./"stack_trace": $body.$_path/]
```

Comment:
I added this so it passes.

This was tricky - dollar signs in the console-result responses are replaced [here](https://github.com/elastic/elasticsearch/blob/632854a4e3a090fbb610a08237551f5cecf40d5d/build-tools-internal/src/main/groovy/org/elasticsearch/gradle/internal/doc/SnippetsTask.groovy#L118). Using source,JSON and replacing the stack_trace did the trick.

Please review to check this is OK!

Reply:
Thanks, I have no idea why that wasn't working for me locally 🤷‍♀️ 🤦‍♀️